blog/2015-02-26-Getting-Started-With_Docker-And-Wildfly.adoc

```diff
@@ -0,0 +1,195 @@
+a798d07859f8        jboss/wildfly:latest   "/opt/jboss/wildfly/   4 seconds ago       Up 3 seconds        8080/tcp, 9990/tcp   backstabbing_poitras   
+....
+
+To kill the container, we'd then execute `docker kill a798d07859f8`, replacing the given hash with your CONTAINER ID. 
+
+
+== What's Missing
+
+The default docker image is lacking a few things for our needs. 
+First, we'd generally like our Wildfly installation to expose our 
+management port if we
+require any management actions or JMX introspection. 
+We might also like SSH access to this machine, if we intend to treat it as a remote system
+and deploy our applications over SCP. 
+We might like to map in 
+some local folders to the running container, if we'd prefer a more quick-and-dirty solution. 
+We could also expose some of the ports from our container 
+as if they were ports on our localhost. 
+
+Depending on our usecase, we may do some or all of the above. 
+
+
+== Externally Managed Local Server With Deployment Folder Mapping
+
+For an externally managed server, where we won't be starting or 
+stopping it from Eclipse, there's a few small changes we can make
+in our command line to launch the image that can make the default
+image much more useful for us. 
+
+=== Viewing the Running Server's Welcome Page
+
+At each launch of our container, the container is granted an IP address. 
+If we were to run our container with the default command as shown above, 
+we'd need to constantly change the IP address of our Server Adapter in JBossTools. 
+To find out the IP of the container, we'd run the following:
+
+....
+[root@localhost docker]$ docker ps
+CONTAINER ID        IMAGE                  COMMAND                CREATED             STATUS              PORTS                                            NAMES
+f70149043400        jboss/wildfly:latest   "/opt/jboss/wildfly/   58 seconds ago      Up 58 seconds       0.0.0.0:8080->8080/tcp, 0.0.0.0:9990->9990/tcp   ecstatic_darwin     
+
+[root@localhost docker]$ docker inspect -f '{{ .NetworkSettings.IPAddress }}' f70149043400
+172.17.0.25
+....
+
+To see the Wildfly welcome page, we could open up any browser and browse to `http://172.17.0.25:8080`.  
+But again, this IP address will change every launch.  To get this to work in tools, 
+we'd need to constantly change our host in our 
+Server Editor, which would quickly become tedious. These commands to get the IP address are also pretty tedious and verbose
+and not at all convenient.  
+
+To make this easier, we can map 
+our container's ports to our localhost ports, exposing those from our container as if they were on localhost. 
+
+ docker run -it -p 8080:8080 -p 9990:9990 jboss/wildfly
+
+In this way, we can now open our browser to `http://localhost:8080` to see the Wildfly welcome page. 
+
+=== Mapping a deployment folder
+
+Since we're not exposing SSH in this example, and we're not using the Management profiles in JBossTools, 
```

Comment:
Its the other way around, something like: "Since the Docker image does not have SSH nor management port enabled we will need to configure JBoss Tools to just use file deployments." ?